demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  flutterPlugin.swift
+//  Runner
+//
+//  Created by Talwinder Kaur on 2022-10-24.
+//
+
+import Flutter
+import UIKit
+import Walletsdk
+
+public class SwiftHelloPlugin: NSObject, FlutterPlugin {
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(name: "HelloPlugin", binaryMessenger: registrar.messenger())
+    let instance = SwiftHelloPlugin()
+    registrar.addMethodCallDelegate(instance, channel: channel)
+  }
+ 
+  public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+      if (call.method == "storeCredentials"){
```

Comment:
One suggestion for future: do not directly put code inside if's in handle function. Better to like this:

```
public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
      if (call.method == "storeCredentials"){
         storeCredentials(args["name"] as! String, result)
      }
}

public func storeCredentials(_ someArg: String, result: @escaping FlutterResult) {
 //Code will be here
}
```